source/l10nUtil.py

```diff
@@ -280,6 +288,97 @@ def uploadTranslationFile(crowdinFilePath: str, localFilePath: str, language: st
 	print("Done")
 
 
+def exportTranslations(outputDir: str, language: str | None = None):
+	"""
+	Export translation files from Crowdin as a bundle.
+	:param outputDir: Directory to save translation files.
+	:param language: The language code to export (e.g., 'es', 'fr', 'de').
+		If None, exports all languages.
+	"""
+
+	# Create output directory if it doesn't exist
+	os.makedirs(outputDir, exist_ok=True)
+
+	client = getCrowdinClient()
+
+	requestData = {
+		# Avoids build warning "Warning: line 392 contained a corrupt empty translation. Using source"
+		"skipUntranslatedStrings": True,
```

Comment:
For our user docs, this will result in a translated xliff with missing translation units. This will then cause markdownTranslate's generateMarkdown command to fail as it can't match an id in the file's skeliton to a translation unit in the file. For example, export translations, and then run  markdownTranslate.py generateMarkdown for hu\userGuide.xliff and the command will fail, because it has nothing translated yet, so its xliff only contains the skeliton. So, I think we need to keep this as False. True you will see some warnings, but I'm pretty usre we already see these. otherwise, the entire file is going to be skipped which is not what we want.

Reply:
```suggestion
		"skipUntranslatedStrings": False,
```